doc/man/8/ceph-objectstore-tool.rst

```diff
@@ -173,8 +174,7 @@ Remove an object (syntax)::
 
 Remove an object (example)::
 
-[root@osd ~]# ceph-objectstore-tool --data-path /var/lib/ceph/osd/ceph-0 --pgid 0.1c '{"oid":"zone_info.default","key":"","snapid":-2,"hash":235010478,"max":0,"pool":11,"namespace":""}' remove
-
+[root@osd ~]# ceph-objectstore-tool --data-path /var/lib/ceph/osd/ceph-0 --pgid ["1.1c",{"oid":"zone_info.default","key":"","snapid":-2,"hash": 235010478,"max":0,"pool":1,"namespace":"","max":0}]
```

Comment:
[root@osd ~]# ceph-objectstore-tool --data-path /var/lib/ceph/osd/ceph-0 '["1.1c",{"oid":"zone_info.default","key":"","snapid":-2,"hash": 235010478,"max":0,"pool":1,"namespace":"","max":0}]' remove

Reply:
Remove --pgid from all object ops examples, syntax for object ops and use sample object JSON:

["1.1c",{"oid":"zone_info.default","key":"","snapid":-2,"hash":235010478,"max":0,"pool":1,"namespace":"","max":0}]

syntax example:
ceph-objectstore-tool --data-path $PATH_TO_OSD $OBJECT remove

Reply:
Can I remove lines 411-414?

They refer to --pgid, and read as follows:

.. option:: --pgid arg
   PG id, mandatory for info, log, remove, export, export-remove, mark-complete, trim-pg-log, and mandatory for apply-layout-settings if --pool is not specified

Reply:
"object ops" in my comment is not referring to --op commands.

So no --pgid still applies to some --op commands. It isn't required for "object ops", which were the ones my original comment was about.